flocker/control/testtools.py

```diff
@@ -341,7 +371,7 @@ def deployment_strategy(
         draw(
             lease_strategy(
                 dataset_id=st.just(dataset_id),
-                node_uuid=st.just(node_uuid)
+                node_id=st.just(node_uuid)
```

Comment:
Was this simply broken before?  It looks like yes but it went unnoticed because with no "stateful" applications there were never any leases.